tests/unit/accordion/accordion_core.js

```diff
@@ -7,15 +7,23 @@ module( "accordion: core", setupTeardown() );
 
 $.each( { div: "#list1", ul: "#navigation", dl: "#accordion-dl" }, function( type, selector ) {
 	test( "markup structure: " + type, function() {
```

Comment:
Isn't this all really still "markup structure" though. If you think we need additional tests to check different option combinations i can add them in the options test file, but i'm not really sure why the changes to this test should not be here?